akka-cluster-typed/src/main/scala/akka/cluster/typed/internal/receptionist/ClusterReceptionist.scala

```diff
@@ -53,20 +61,46 @@ private[typed] object ClusterReceptionist extends ReceptionistBehaviorProvider {
     def apply(map: ORMultiMap[ServiceKey[_], ActorRef[_]]): ServiceRegistry = TypedORMultiMap[ServiceKey, ActorRef](map)
   }
 
+  private case object RemoveTick
+
   def behavior: Behavior[Command] = clusterBehavior
-  val clusterBehavior: Behavior[Command] = ReceptionistImpl.init(clusteredReceptionist())
+  val clusterBehavior: Behavior[Command] = ReceptionistImpl.init(ctx ⇒ clusteredReceptionist(ctx))
+
+  object ClusterReceptionistSettings {
+    def apply(system: ActorSystem[_]): ClusterReceptionistSettings =
+      apply(system.settings.config.getConfig("akka.cluster.typed.receptionist"))
+
+    def apply(config: Config): ClusterReceptionistSettings = {
+      val writeTimeout = 5.seconds // the timeout is not important
```

Comment:
Does "not important" mean "not used"? Or, we are not waiting for confirmation anyway?

Reply:
we are not waiting for confirmation, using a higher consistency level than "local" would only be a way to speed up replication